CMakeLists.txt

```diff
@@ -1157,16 +1157,16 @@ endif (UNIX)
 #   Release => -O3 -NDEBUG
```

Comment:
I'm confused by the i#5266 comment. Did this solution work for you? If not, I  notice that we reset the `CFLAGS` env var here:
https://github.com/DynamoRIO/dynamorio/blob/27193828261be587d06bcbbca3eabc49726d441f/suite/runsuite.cmake#L413
I haven't verified if this is the cause; can you check?

Reply:
What comment? Yes, this solution works perfectly for me.
I don't think anything in suite/runsuite.cmake will anyhow affect things. I am building just libdrdecode.a not the tests.